vision/benchmarks/accuracy/bench_module.py

```diff
@@ -352,26 +352,33 @@ def bench_module_accuracy(folder: str, quiet_output: bool = False, draw_centers:
                         "CRASH:",crash,
                     )
                 
+
+                marked_image = np.copy(image)
+
                 # if enabled and available, draw circle on center
                 if draw_centers:
-                    image_copy = np.copy(image)
                     cv2.circle(
-                        image_copy, 
-                        (center[0], center[1]), 
-                        20, 
-                        (0, 0, 255), 
-                        3
+                        img=marked_image, 
+                        center=(center[0], center[1]), 
+                        radius=20, 
+                        color=(0, 0, 255), 
+                        thickness=3
                     )
                     cv2.circle(
-                        image_copy, 
-                        (center[0], center[1]), 
-                        1, 
-                        (0, 0, 255), 
-                        2
+                        img=marked_image, 
+                        center=(center[0], center[1]), 
+                        radius=1, 
+                        color=(0, 0, 255), 
+                        thickness=2
                     )
 
-                    cv2.imwrite(os.path.join(DRAW_CENTERS_DIR,file), image_copy)
+                # if enabled and available, draw hough circles
+                if draw_hough_circles and tester.location.circles is not None:
```

Comment:
Mateusz also added a feature to draw the circles. His uses a function in the location class, so is probably more robust. Make sure you resolve that in the merge conflict properly so that only one version of the draw circles is in the benchmark.